Classes/Service/VarnishBanService.php

```diff
@@ -151,5 +146,24 @@ private function domainsToArray($domains = null): array
         $domains = $domains ?? [];
         return is_array($domains) ? $domains : [$domains];
     }
+
+    private function prepareVarnishUrls(): array
+    {
+        $varnishUrls = $this->settings['varnishUrl'] ?? ['http://127.0.0.1'];
+
+        if (is_string($varnishUrls)) {
+            if (strpos($varnishUrls, ',') > 0) {
```

Comment:
```suggestion
            $varnishUrls = array_filter(explode(',', $varnishUrls));
```
This will remove empty entries and cast a string with only on string to array. Then you can get rid of the condition.